src/store/asana/index.ts

```diff
@@ -425,6 +467,20 @@ export default {
         commit("releaseTask", task);
       });
     },
+    updateTaskTags({ commit }, taskAndSectionId: TaskAndSectionId): void {
+      const originalTagIds = taskAndSectionId.task.tags.map(tag => tag.gid)
+      const tempTagIds = taskAndSectionId.tempTags.map(tag => tag.gid)
+
+      const removedTags = originalTagIds.filter(tag => !tempTagIds.includes(tag))
```

Comment:
Can you please rename the closure variable to be tagId? That way it is named different to the ones above (since they are different things)?